src/lib/styles/mixins.sass

```diff
@@ -31,3 +31,10 @@
   border-radius: var(--radius)
   box-shadow: 1px 1px 5px 0 rgba(0, 0, 0, 0.02), 1px 1px 15px 0 rgba(0, 0, 0, 0.03)
   transition: transform var(--transition-normal), background-color var(--transition-normal), box-shadow var(--transition-slow)
+
+@mixin image-hover-animation
```

Comment:
maybe rename to something shorter? `hover-scale` or something like that? technically it's not bound to an imge like that

Reply:
Done